metrics/metrics.go

```diff
@@ -151,3 +157,17 @@ func Add(id MetricID, value MetricValue) {
 // If these assumptions change, we can address that by regularly calling AddSlice() with
 // an empty slice. Code can be found in commit 1d01d1ff841891010afaf8d64d4c21a05f19d168
 // and earlier.
+
+// GetDefinitions returns the metric definitions from the embedded metrics.json file.
+func GetDefinitions() ([]MetricDefinition, error) {
+	var definitions []MetricDefinition
+
+	dec := json.NewDecoder(bytes.NewReader(metricsJSON))
+	dec.DisallowUnknownFields()
+
+	err := dec.Decode(&definitions)
+	if err != nil {
+		return nil, err
+	}
+	return definitions, nil
```

Comment:
NIT: This could be simplified:

```suggestion
	return definitions, dec.Decode(&definitions)
```

Reply:
IIUC, unfortunately this linter rule disallows this: https://go-critic.com/overview.html#evalorder